curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553f4f66099802c6ae94613.md

```diff
@@ -16,13 +18,11 @@ Give `message` and `offset` values, by passing `text` and `shift` as arguments t
 You should pass `text` and `shift` as the arguments to your function call by including them inside the parentheses. Don't forget to separate the arguments with a comma.
 
 ```js
-assert.match(code, /^caesar\s*\(\s*text\s*,\s*shift\s*\)/m)
-```
-
-Your code contains invalid syntax and/or invalid indentation.
```

Comment:
We might consider to get rid of those final tests that run the code and substitute them all, but I would focus on the instructions for now.